cordova/config/config.template

```diff
@@ -1,11 +1,11 @@
 <?xml version='1.0' encoding='utf-8'?>
 <widget id="org.montelibero.solar" version="$PACKAGE_VERSION" android-versionCode="$ANDROID_VERSIONCODE" ios-CFBundleVersion="$IOS_BUNDLE_VERSION" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools">
-    <name>Solar</name>
+    <name>MTL Solar</name>
     <description>
         Wallet for the Stellar payment network by SatoshiPay.
     </description>
-    <author email="hello@solarwallet.io" href="https://solarwallet.io/">
-        SatoshiPay Ltd
+    <author email="solar@montelibero.org" href="https://solar.montelibero.org/">
+        Montelibero Solar Team
     </author>
     <license>GPL-3.0</license>
```

Comment:
MIT

Reply:
UPD: "license" block could be removed. It doesn't make sense, because it's not defined in [docs](https://cordova.apache.org/docs/en/12.x/config_ref/)